lindeck-frontend/src/common/classes/SessionObject.jsx

```diff
@@ -0,0 +1,16 @@
+
+export default class MySession {
+    username = ""
+    myToken = "" // ???
+    isActive = false;
+
+    constructor(myUserName, myToken) {
+        this.username = myUserName || this.username
+        this.myToken = myToken || this.username
```

Comment:
?

Reply:
set this.value to this.valse if myValue is null

Reply:
You set this.myToken as **username** when myToken is null, why? And please don not use "my" in variable names.